app/src/main/res/layout/media_layout.xml

```diff
@@ -16,10 +16,27 @@
       tools:visibility="visible"
       />
 
-  <VideoView
-      android:id="@+id/video_item"
+  <FrameLayout
+      android:id="@+id/editorial_video_thumbnail_container"
       android:layout_width="match_parent"
-      android:layout_height="match_parent"
+      android:layout_height="200dp"
```

Comment:
this should be match parent (I know it will be the same in this case) (I wouldn’t say to do this if it was the only thing)